settings.gradle

```diff
@@ -180,8 +180,11 @@ include "beam-sdks-python-container"
 project(":beam-sdks-python-container").dir = file("sdks/python/container")
 include "beam-sdks-python-container-py3"
 project(":beam-sdks-python-container-py3").dir = file("sdks/python/container/py3")
+// Temporary version that will be deleted once we migrate to using the published version.
 include "beam-vendor-grpc-v1_13_1"
```

Comment:
These two projects have the exact same name aside for `v1_13_1` vs `1_31_1`. This seems like a recipe for confusion. Is there any better identifier that could be used to differentiate them?

Reply:
v1_13_1 is the one currently being used for vendoring gRPC for use in the project till the 1_13_1 is published and then v1_13_1 will be deleted. This is reason why I mentioned I could get it submitted but its a little bit of a hassle.

Reply:
Got it, thanks